CHANGELOG.md

```diff
@@ -4,6 +4,12 @@
 
 Not yet released; provisionally v2.0.0 (may change).
 
+### GetLatestSignedLogRoot With Consistency Proof
+
+`GetLatestSignedLogRoot` in LogServer also returns a consistency proof if
+possible, in addition to the latest signed log root. This reduces the number
```

Comment:
```suggestion
possible, in addition to the latest signed log root. This reduces the number of
```

Reply:
Also, should mention that it'll only do this if the new `first_tree_size` field is set in `GetLatestSignedLogRootRequest`.